TestForwardsTransactionWithFaultySigner hangs

```
=== RUN   TestForwardsTransactionWithFaultySigner
scvg0: inuse: 2, idle: 61, sys: 63, released: 0, consumed: 63 (MB)
scvg1: 61 MB released
scvg1: inuse: 2, idle: 61, sys: 63, released: 61, consumed: 2 (MB)
scvg2: inuse: 2, idle: 61, sys: 63, released: 61, consumed: 2 (MB)
scvg3: inuse: 2, idle: 61, sys: 63, released: 61, consumed: 2 (MB)
scvg4: inuse: 2, idle: 61, sys: 63, released: 61, consumed: 2 (MB)
scvg5: inuse: 2, idle: 61, sys: 63, released: 61, consumed: 2 (MB)
panic: test timed out after 20m0s

goroutine 5 [running]:
testing.(*M).startAlarm.func1()
	/usr/local/go/src/testing/testing.go:1296 +0x13a
created by time.goFunc
	/usr/local/go/src/time/sleep.go:172 +0x52

goroutine 1 [chan receive]:
testing.(*T).Run(0xc00019c700, 0xb5c713, 0x27, 0xb6aa20, 0xc000159c01)
	/usr/local/go/src/testing/testing.go:879 +0x692
testing.runTests.func1(0xc00019c700)
	/usr/local/go/src/testing/testing.go:1119 +0xa9
testing.tRunner(0xc00019c700, 0xc000159d88)
	/usr/local/go/src/testing/testing.go:827 +0x163
testing.runTests(0xc000122540, 0x11d11e0, 0x22, 0x22, 0xc0001bc448)
	/usr/local/go/src/testing/testing.go:1117 +0x4ef
testing.(*M).Run(0xc00019a380, 0x0)
	/usr/local/go/src/testing/testing.go:1034 +0x2ef
main.main()
	_testmain.go:108 +0x222

goroutine 43 [chan receive]:
testing.(*T).Parallel(0xc00019cc00)
	/usr/local/go/src/testing/testing.go:732 +0x3be
github.com/orbs-network/orbs-network-go/services/transactionpool.TestCommittedTransactionPoolClearsOldTransactions(0xc00019cc00)
	/go/src/github.com/orbs-network/orbs-network-go/services/transactionpool/committed_transaction_pool_test.go:22 +0x39
testing.tRunner(0xc00019cc00, 0xb6aa08)
	/usr/local/go/src/testing/testing.go:827 +0x163
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:878 +0x65a

goroutine 67 [chan send]:
github.com/orbs-network/orbs-network-go/services/transactionpool.(*transactionForwarder).submit(0xc0002541b0, 0xc0001d1d28, 0x1, 0x1)
	/go/src/github.com/orbs-network/orbs-network-go/services/transactionpool/forward_transactions.go:92 +0x9b
github.com/orbs-network/orbs-network-go/services/transactionpool.TestForwardsTransactionWithFaultySigner.func1(0xbe0800, 0xc000250040, 0xc00025a0a0)
	/go/src/github.com/orbs-network/orbs-network-go/services/transactionpool/forward_transactions_test.go:139 +0x635
github.com/orbs-network/orbs-network-go/test.WithConcurrencyHarness(0xbe31e0, 0xc000268000, 0xc0001d6f58)
	/go/src/github.com/orbs-network/orbs-network-go/test/concurrency.go:37 +0x5e6
github.com/orbs-network/orbs-network-go/services/transactionpool.TestForwardsTransactionWithFaultySigner(0xc000268000)
	/go/src/github.com/orbs-network/orbs-network-go/services/transactionpool/forward_transactions_test.go:121 +0x68
testing.tRunner(0xc000268000, 0xb6aa20)
	/usr/local/go/src/testing/testing.go:827 +0x163
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:878 +0x65a

goroutine 68 [chan send]:
github.com/orbs-network/orbs-network-go/services/transactionpool.(*transactionForwarder).submit(0xc0002541b0, 0xc000258150, 0x2, 0x2)
	/go/src/github.com/orbs-network/orbs-network-go/services/transactionpool/forward_transactions.go:92 +0x9b
github.com/orbs-network/orbs-network-go/services/transactionpool.(*transactionForwarder).drainQueueAndForward(0xc0002541b0, 0xbe08c0, 0xc0002bc000)
	/go/src/github.com/orbs-network/orbs-network-go/services/transactionpool/forward_transactions.go:141 +0x114f
github.com/orbs-network/orbs-network-go/services/transactionpool.(*transactionForwarder).start.func1()
	/go/src/github.com/orbs-network/orbs-network-go/services/transactionpool/forward_transactions.go:118 +0x2df
github.com/orbs-network/orbs-network-go/vendor/github.com/orbs-network/govnr.tryOnce(0xbdc8c0, 0xc0002580e0, 0xc00026c080)
	/go/src/github.com/orbs-network/orbs-network-go/vendor/github.com/orbs-network/govnr/panic.go:19 +0x69
github.com/orbs-network/orbs-network-go/vendor/github.com/orbs-network/govnr.Forever.func1(0xc000250080, 0xbdc8c0, 0xc0002580e0, 0xc00026c080, 0xbe0800, 0xc000250040)
	/go/src/github.com/orbs-network/orbs-network-go/vendor/github.com/orbs-network/govnr/forever.go:76 +0x77
created by github.com/orbs-network/orbs-network-go/vendor/github.com/orbs-network/govnr.Forever
	/go/src/github.com/orbs-network/orbs-network-go/vendor/github.com/orbs-network/govnr/forever.go:72 +0x18a
FAIL	github.com/orbs-network/orbs-network-go/services/transactionpool	1200.115s
+ exit 1
```

https://circleci.com/gh/orbs-network/orbs-network-go/18741#tests/containers/0

--- a/services/transactionpool/committed_transaction_pool_test.go
+++ b/services/transactionpool/committed_transaction_pool_test.go
@@ -19,8 +19,6 @@ import (
 )
 
 func TestCommittedTransactionPoolClearsOldTransactions(t *testing.T) {
-	t.Parallel()
-
 	test.WithContext(func(ctx context.Context) {
 		p := NewCommittedPool(func() time.Duration { return time.Second }, metric.NewRegistry())
 		ctrlRand := rand.NewControlledRand(t)

--- a/services/transactionpool/forward_transactions_test.go
+++ b/services/transactionpool/forward_transactions_test.go
@@ -131,18 +131,16 @@ func TestForwardsTransactionWithFaultySigner(t *testing.T) {
 		harness.Supervise(txForwarder)
 
 		tx := builders.TransferTransaction().Build()
-		anotherTx := builders.TransferTransaction().Build()
 
 		gossip.When("BroadcastForwardedTransactions", mock.Any, mock.Any).Return(nil, nil).Times(0)
 
 		txForwarder.submit(tx)
-		txForwarder.submit(anotherTx)
 
 		require.NoError(t, test.ConsistentlyVerify(cfg.TransactionPoolPropagationBatchingTimeout()*2, gossip), "mocks were not called as expected")
 
-		oneBigHash, _, _ := HashTransactions(tx, anotherTx)
+		oneBigHash, _, _ := HashTransactions(tx)
 		sig, _ := signer.Sign(ctx, oneBigHash)
-		expectTransactionsToBeForwarded(gossip, cfg.NodeAddress(), sig, tx, anotherTx)
+		expectTransactionsToBeForwarded(gossip, cfg.NodeAddress(), sig, tx)
 
 		signer.Reset()
 		signer.When("Sign", mock.Any, mock.Any).Return(sig, nil).Times(1)

